### PR TITLE
Update jsonschema-rs & error messages formatting

### PR DESCRIPTION
**Pull request by driv3r**

### CODE REVIEW

File: spec/rusty_json_schema_spec.rb [hunk 1]
```diff
@@ -63,9 +63,9 @@
 
       it "returns validation errors" do
         expect(validator.validate(event)).to eq [
-          "'\"rusty\"' is not of type 'number'",
-          "'1' is not of type 'string'",
-          "'baz' is a required property"
+          %("rusty" is not of type "number" at /bar),
```
Comment (mobileka) [inline spec/rusty_json_schema_spec.rb]: Just a matter of taste but I would format this as follows:

`path "foo": "rusty" is not of type "number"`

I think that it's easier for humans to read this as most of them, in my opinion, think in `property` -> `value` and not `value` -> `property` :)

**Review by mobileka — APPROVED** [review]

File: spec/rusty_json_schema_spec.rb (hunk 1 above)
Comment (driv3r) [reply spec/rusty_json_schema_spec.rb]: makes sense! I'll set it to `path "/foo": ...` and `at root: ...`, does it make sense? @mobileka

Comment (driv3r) [reply spec/rusty_json_schema_spec.rb]: or even better `path "/": ...`

Comment (mobileka) [reply spec/rusty_json_schema_spec.rb]: Yup, I would also vote for `"/"` 👍

**Pull request merged**